### PR TITLE
[4.0] Fix installation js for messages

### PR DESCRIPTION
**Pull request by Fedik**
Pull Request for Issue #35170 .

### Summary of Changes

Correct call of `Joomla.renderMessage()`

### Testing Instructions
Please follow #35170

### CODE REVIEW

File: installation/template/js/remove.js [hunk 1]
```diff
@@ -73,11 +73,11 @@ Joomla.deleteJoomlaInstallationDirectory = function (redirectUrl) {
             const successresponse = JSON.parse(response);
             if (successresponse.error === true) {
                 if (successresponse.messages) {
-                    Joomla.renderMessages(successresponse.messages, '#system-message-container');
+                    Joomla.renderMessages(successresponse.messages);
```
Comment (wilsonge) [inline installation/template/js/remove.js:76]: This needs to have the message in an object too with an error key

Comment (Fedik) [reply installation/template/js/remove.js:76]: it should defaulting there:
https://github.com/joomla/joomla-cms/blob/faf9352ba1a5f822cdf73bb4e1dfd9fa39340fc3/build/media_source/system/js/messages.es6.js#L22-L24

That why I removed it here.

Or you got any bug with it?

Comment (wilsonge) [inline installation/template/js/remove.js:76]:
```suggestion
                    Joomla.renderMessages({error: [successresponse.messages]});
```

Comment (wilsonge) [reply installation/template/js/remove.js:76]: I mean I *think* this is required the same as we have below? Sorry I wasn't clear above!

**Comment by wilsonge** [comment]
Thanks!

File: installation/template/js/remove.js (hunk 1 above)
Comment (Fedik) [reply installation/template/js/remove.js:76]: hm, here we actually have an object of messages?

**Comment by PhilETaylor** [comment]
Seems to work... 
Screenshot 2021-08-17 at 10 22 48

**Pull request merged**